tests/manifold/spherical_manifold_06.cc doesn't currently compile

I guess that's one for you, @luca-heltai :-)

```
FAILED: /usr/bin/c++   -DDEBUG -DSOURCE_DIR=\"/node/bangerth/trunk/dealii/tests/manifold\" -I/node/bangerth/trunk/build-64bit/include -I/node/bangerth/trunk/dealii/include -I/node/bangerth/trunk/dealii/bundled/umfpack/UMFPACK/Include -I/node/bangerth/trunk/dealii/bundled/umfpack/AMD/Include -I/node/bangerth/trunk/dealii/bundled/boost-1.56.0/include -I/node/bangerth/trunk/dealii/bundled/muparser_v2_2_4/include -I/apps/openmpi/1.6.5/include -I/node/bangerth/bin/petsc-3.6.1-mpi/x86_64-64bit/include -I/node/bangerth/bin/trilinos-11.6.1/include -I/node/bangerth/bin/petsc-3.6.1-mpi/include -I/node/bangerth/bin/p4est-1.1/FAST/include -I/node/bangerth/bin/slepc-3.6.2-mpi/x86_64-64bit/include -I/node/bangerth/bin/slepc-3.6.2-mpi/include    -pedantic -fpic -Wall -Wextra -Wpointer-arith -Wwrite-strings -Wsynth -Wsign-compare -Wswitch -Woverloaded-virtual -Wno-long-long  -std=c++0x -B/w/bangerth/share/software/bin/ld -O0 -ggdb -Wa,--compress-debug-sections -MMD -MT CMakeFiles/spherical_manifold_06.debug.dir/spherical_manifold_06.cc.o -MF "CMakeFiles/spherical_manifold_06.debug.dir/spherical_manifold_06.cc.o.d" -o CMakeFiles/spherical_manifold_06.debug.dir/spherical_manifold_06.cc.o -c /node/bangerth/trunk/dealii/tests/manifold/spherical_manifold_06.cc
/node/bangerth/trunk/dealii/tests/manifold/spherical_manifold_06.cc:24:41: fatal error: deal.II/fe/mapping_manifold.h: No such file or directory
compilation terminated.
```

By accident, I think you include a file that isn't part of deal.II yet.

--- a/tests/manifold/spherical_manifold_06.cc
+++ b/tests/manifold/spherical_manifold_06.cc
@@ -21,7 +21,6 @@
 #include <deal.II/grid/tria.h>
 #include <deal.II/grid/grid_generator.h>
 #include <deal.II/fe/mapping_q_generic.h>
-#include <deal.II/fe/mapping_manifold.h>
 #include <deal.II/fe/fe_values.h>
 #include <deal.II/grid/manifold_lib.h>
 

